vuepress/docs/next/docs/ecr/ecr-bundle-details.md

```diff
@@ -248,7 +247,7 @@ Here is an example of a widget descriptor:
       </html>
 
 ## Page
-This descriptor enables a page to be created and published via a bundle. Status can be `published` or `draft`. The widgets section can be used to fully configure a page layout.
+This descriptor enables a page to be created and published via a bundle. Page status can be `published` or `draft`. The widgets section can be used to fully configure a page layout.
```

Comment:
"The widgets section can be used to fully configure a page layout."  Can we make widgets singular, it reads like it should be possessive... maybe just me